addons/web/static/src/js/core/ajax.js

```diff
@@ -148,6 +148,28 @@ function loadCSS(url) {
     }
 }
 
+/**
+ * Load css asynchronously: fetch it from the url parameter
+ * and add it inline in <head>.
+ * Tries to load the same url only once by assigning it an id (url with
+ * non-word characters removed).
+ *
+ * @param {String} url of the css to be fetched
+ * @returns {Deferred} resolved when the css has been appended.
+ */
+function loadCSSAsync(url) {
```

Comment:
remove

Reply:
note that there is a loadCSS function

Reply:
ged suggested rightfully that it would be a bad practice to always load all css from all the banners. I added this mechanism to be able to know when the css was loaded and prevent displaying unstyled content. I removed this in the current build so you can judge if displaying the unstyled onboarding panel a few miliseconds is ok or not.
Simply open and close one of the steps a few times to view and experience what I'm describing.

Reply:
After a discussion with antony --> Don't remove.